packages/app/src/app/overmind/effects/api/index.ts

```diff
@@ -680,8 +680,15 @@ export default {
       `/teams/${teamId}/customer_portal?return_path=${return_path}`
     );
   },
-  removeBranchFromRepository(owner: string, repo: string, branch: string) {
-    return api.delete(`/beta/sandboxes/github/${owner}/${repo}/${branch}`);
+  removeBranchFromRepository(
+    workspaceId: string,
+    owner: string,
+    repo: string,
+    branch: string
+  ) {
```

Comment:
Maybe we should pass an object instead of multiple parameters.

Reply:
This might be replaced with a mutation, I'll check how it works now and transition if needed, then the params can be better handled